scene/2d/audio_stream_player_2d.cpp

```diff
@@ -391,7 +399,17 @@ int AudioStreamPlayer2D::get_max_polyphony() const {
 	return max_polyphony;
 }
 
+void AudioStreamPlayer2D::set_panning_strength(float p_panning_strength) {
+	panning_strength = p_panning_strength;
+}
+
+float AudioStreamPlayer2D::get_panning_strength() const {
+	return panning_strength;
+}
+
 void AudioStreamPlayer2D::_bind_methods() {
+	GLOBAL_DEF("audio/general/2d_panning_strength", 0.5f);
```

Comment:
I think it's better to move the setting declaration to ProjectSettings class, so it's better centralized. This also lets you define a range for the setting:

https://github.com/godotengine/godot/blob/92615be68cddfc30b943f5eea942cc3c22b84bb6/core/config/project_settings.cpp#L1216-L1217